src/test/stellar/test-purchase-nft.ts

```
@@ -4,92 +4,553 @@ import { purchaseNFT } from '@/services/purchase-nft.service';
+        success: false,
+        error: `Balance verification failed: ${error instanceof Error ? error.message : String(error)}`
+      };
+    }
+  }
+
+  private async cleanup(): Promise<void> {
+    try {
+
+    } catch (error) {
+      console.warn(`Cleanup warning: ${error instanceof Error ? error.message : String(error)}`);
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)